src/index.html

```diff
@@ -77,9 +77,11 @@
             </div>
```

Comment:
FYI, I've used https://github.com/gregberge/svgr to convert SVG's to react components before, it's a neat tool

Reply:
Thanks, will check it out.